docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc

```diff
@@ -23,26 +19,31 @@ For Maven:
 </dependency>
 ----
 
-`quarkus-opentelemetry` extension already includes the default OTLP exporter. To deactivate that default exporter and only have this Azure extension exporting data, please set the following property in the `application.properties`:
+You also need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string [in the top right corner](https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string).
+
+You can set the connection string:
+* In the `application.properties` file:
 
 [source]
 ----
-quarkus.opentelemetry.tracer.exporter.otlp.enabled=false
+quarkus.otel.azure.applicationinsights.connection.string=your_connection_string
 ----
 
-Enable the Azure export by adding following properties in the `application.properties`:
+* Or as an environment variable: `QUARKUS_OTEL_AZURE_APPLICATIONINSIGHTS.CONNECTION_STRING=your_connection_string`
+
+You can disable the Azure export in this way:
 
 [source]
 ----
 quarkus.opentelemetry.tracer.exporter.azure.enabled=true
```

Comment:
To disable it should be 
`quarkus.opentelemetry.tracer.exporter.azure.enabled=false`